### PR TITLE
feat: added new PIOS and Debian build/running guide

### PR DESCRIPTION
**Pull request by thobbsinteractive**

### CODE REVIEW

File: README-PI-DEBIAN.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+     3. next to the `remc2` binary
+  
+  Run the `remc2` executable in install directory
+  ```bash
+  cd /Documents/repos/magic-carpet-2-hd/build/${BUILDTYPE}/inst/bin
+  ./remc2
+  ```
+
+### Notes
+Currently the Game will not load the first level, it fails with a `Segment Fault`
```
Comment (GrimSqueaker) [inline README-PI-DEBIAN.md:57]: Hmm, I guess that on Debian x86/x64 the game will not segfault.

File: README-PI-DEBIAN.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+### Building on Pi OS/Debian
+
```
Comment (GrimSqueaker) [inline README-PI-DEBIAN.md:2]: I'm not sure if we should call this document like that.
Most of the info here is generic Linux information.
The only specific info for Debian-based distibutions (Raspberry Pi OS, Ubuntu, Linux Mint, ...) are the `sudo apt install` commands and some wording.
The problem I see here, is that we now have duplicated documentation and need to maintain it in two places. Maybe it is better to include it in the main readme.md with some wording to explain that the commands are examples for Debian-based distributions.

Comment (thobbsinteractive) [reply README-PI-DEBIAN.md:2]: okay. I will try and merge it with the main readme

File: README-PI-DEBIAN.md (hunk 1 above)
Comment (thobbsinteractive) [reply README-PI-DEBIAN.md]: who knows. It is a Segmentation Fault. I am looking into it.

**Review by GrimSqueaker — APPROVED** [review]
Great :-)

**Pull request merged**